controllers/userController.js

```diff
@@ -175,17 +175,71 @@ const currentUser = async (req, res, next) => {
   ApiResponse.result(res, { user: data }, httpStatusCodes.OK);
 };
 
+const updateUser = async (req, res, next) => {
+  const { firstName, lastName, phone } = req.body;
+
+    const user = await Users.updateOne(
+      { _id: req.params.id },
+      { $set: {firstName, lastName, phone} }
+    );
+
+    ApiResponse.result(res, { user }, httpStatusCodes.OK);
+};
+
 //logout user
 const logoutUser = (req, res, next) => {
   res.cookie("accessToken", "", { maxAge: 0 });
   ApiResponse.result(res, { status: "User logged out successfully" }, httpStatusCodes.OK);
 };
+const changePassword = async (req, res, next) => {
+  const { oldpassword, newpassword, confirmpassword } = req.body;
+   if (!oldpassword || !newpassword || !confirmpassword){
```

Comment:
we don't send confirm password on backend. Its just a validation thing for frontend